README.md

```diff
@@ -91,6 +91,12 @@ dependencies.
 
 ### Developing
 
+#### Updating dependencies
+
+Run `bazel run :go -- get -u DEPENDENCY && bazel run :gazelle-update-repos`
```

Comment:
I think we may want to remove the `-u` here. That tells the toolchain to grab the dependency but also upgrade any related deps, which may not always be the expected result.